Don't persist custom endpoints to the config file (#22)
giantswarm/gscliauth: CHANGELOG.md

## [Unreleased]

- Increase resiliency using custom endpoints in `ChooseEndpoint`, `ChooseScheme`, `ChooseToken` and `SetProvider`

> - When using custom endpoints, unknown endpoint configurations will not get persisted into the config file any more

Reviewer: ```suggestion
- When using custom endpoints, unknown endpoint configurations will not be persisted in the config file anymore
```

> - When using custom endpoints, the custom endpoint will not be selected any more

Reviewer: ```suggestion
- When using custom endpoints, the custom endpoint will not be selected anymore
```
